lib/Onyx.js

```diff
@@ -766,7 +814,7 @@ function connect(mapping) {
             // since there are none matched. In withOnyx() we wait for all connected keys to return a value before rendering the child
             // component. This null value will be filtered out so that the connected component can utilize defaultProps.
             if (matchingKeys.length === 0) {
-                sendDataToConnection(mapping, null);
+                sendDataToConnection(mapping, null, undefined, false);
```

Comment:
This update should not be batched because we rely on the value being set immediatly for defaultProps

Reply:
I think it would be good to add this as a comment. Can you write a brief line or two that also explains:

- why we avoid the batch in this case
- what bad thing will happen if we do batch it
- how it relates to `defaultProps`